garage/tf/core/layers.py

```diff
@@ -1052,7 +1052,7 @@ class GRULayer(Layer):
     Reset gate:        r(t) = f_r(x(t) @ W_xr + h(t-1) @ W_hr + b_r)
     Update gate:       u(t) = f_u(x(t) @ W_xu + h(t-1) @ W_hu + b_u)
     Cell gate:         c(t) = f_c(x(t) @ W_xc + r(t) * (h(t-1) @ W_hc) + b_c)
-    New hidden state:  h(t) = (1 - u(t)) * h(t-1) + u_t * c(t)
+    New hidden state:  h(t) = (1 - u(t)) * c(t) + u_t * h(t-1)
```

Comment:
I think the original update is correct?

Reply:
Original update is not the same as the origin paper.

Reply:
Please check? I think the old one is consistent with the paper and the wikipedia.

Reply:
emm I think so. TensorFlow is not, in this case. Since we are using TensorFlow GRUCell, I wanted to make the existing and new `GRU` give the same output. Right now they are like inverse of each other, one of them is learning `x` while another one is learning `1-x`.

Reply:
If we don't have to care about making them having the same output, I will stick with the original implementation.

Reply:
They are both okay to me and they can both be seen as the original paper. They used the tf notation in 1406.1078v3 then used our old notation in 1412.3555v1. If we need to make sure the new one and the old one has the same implementation, we should probably be consistent with tf implementation. @ryanjulian What do you think?

Reply:
I talked with Ryan and he agreed we should stick with tf implementation.